storage/operation/payload.go

```diff
@@ -0,0 +1,25 @@
+package operation
```

Comment:
Maybe these functions could live in a file like `execution_fork_evidence.go`? `payload.go` is an unintuitive name to me.